cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/pcluster_slurm_config_generator.py

```diff
@@ -74,8 +75,12 @@ def generate_slurm_config_files(
             )
         is_default_queue = False
 
-    # Generate slurm_parallelcluster.conf and slurm_parallelcluster_gres.conf
-    for template_name in ["slurm_parallelcluster.conf", "slurm_parallelcluster_gres.conf"]:
+    # Generate slurm_parallelcluster.conf, slurm_parallelcluster_gres.conf and slurm_parallelcluster_cgroup.conf
```

Comment:
Just out of my curiosity: what is the difference between the three configuration files?